src/strategy/mod.rs

```diff
@@ -148,6 +157,12 @@ impl UpdateStrategy {
         let periodic = StrategyPeriodic::new(cfg)?;
         Ok(UpdateStrategy::Periodic(periodic))
     }
+
+    /// Build a new "filesystem" strategy.
+    fn new_filesystem() -> Self {
+        let filesystem = StrategyFilesystem::default();
+        UpdateStrategy::Filesystem(filesystem)
```

Comment:
```suggestion
        UpdateStrategy::Filesystem(Default::default())
```
would probably work too.

Reply:
hmm, I was trying to keep it consistent with how the other strategies are created.